modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -152,22 +193,57 @@ function inope_policy_review_form_submit($form, &$form_state) {
 
   // If 'I have reviewed this policy and approved it for use' is selected.
   if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
+
+    // Only administrators should be able to approve notes.
+    if (in_array('administrator', $account->roles)) {
+
+      // Run an update query on the policy review table to set the note_status
+      // as approved where the fields are equal to the current node and are
+      // unapproved.
+      $updated_records = db_update('policy_review')
+        ->fields(array(
+          'note_status' => 1,
+        ))
+        ->condition('nid', $node->nid, '=')
+        ->condition('note_status', 0, '=')
+        ->execute();
+
+      drupal_set_message(t('Policy approved for use @number records updated', array(
+        '@number' => $updated_records,
+      )));
+    }
+    // Staff will receive this warning message.
+    else {
+      drupal_set_message(t('Please contact your administrator for approval.'),
+        'warning');
+    }
+
   }
   // If 'I have reviewed this policy and would like to see changes' is selected.
   elseif ($form_state['values']['inope_policy_review_status_radio'] == 1) {
+
+    // Make code more legible by setting a variable here.
+    $text = $form_state['values']['inope_policy_review_changes_requested_textarea'];
+
+    // Run an insert query on the policy review table for the new note.
+    $policy_review_id = db_insert('policy_review')
+      ->fields(array(
+        'nid' => $node->nid,
+        'vid' => $node->vid,
+        'uid' => $account->uid,
+        'note_status' => 0,
+        'note_submitted' => time(),
+        'note' => $text,
+      ))
+      ->execute();
+
+    // Thank the user for their request.
     drupal_set_message(
-      t('User : @userid : @username submitted the following change request:', array(
-        '@userid' => $account->uid,
+      t('Thank you @username for submitting the following change request:', array(
         '@username' => $account->name,
       )));
 
-    drupal_set_message($form_state['values']['inope_policy_review_changes_requested_textarea']);
-
-    drupal_set_message(t('Node ID : @nodeid', array('@nodeid' => $node->nid)));
-    drupal_set_message(t('Node VID : @nodevid', array('@nodevid' => $node->vid)));
-    // TODO: put this into the database.
+    drupal_set_message($text);
```

Comment:
probably want to validate this text before you repost it. Using user submitted material straight up is probably not a great idea -- even in an intranet -- they may copy something or something else intentionally so -- while I think we can generally assume that form monkeying is not in place, I think this should probably be scrubbed